src/Frontend/Themes/Bootstrap4/src/Layout/Sass/_bootstrap-variables.scss

```diff
@@ -75,40 +77,57 @@ $light:         $gray-100;
 $dark:          $gray-800;
 
 $theme-colors: ();
-$theme-colors: map-merge((
-  "primary":    $primary,
-  "secondary":  $secondary,
-  "success":    $success,
-  "info":       $info,
-  "warning":    $warning,
-  "danger":     $danger,
-  "light":      $light,
-  "dark":       $dark
-), $theme-colors);
+$theme-colors: map-merge(
+  (
+    "primary":    $primary,
+    "secondary":  $secondary,
+    "success":    $success,
+    "info":       $info,
+    "warning":    $warning,
+    "danger":     $danger,
+    "light":      $light,
+    "dark":       $dark
+  ),
+  $theme-colors
+);
 // stylelint-enable
 
 // Set a specific jump point for requesting color jumps
 $theme-color-interval:      8%;
 
 // The yiq lightness value that determines when the lightness of color changes from "dark" to "light". Acceptable values are between 0 and 255.
-$yiq-contrasted-threshold: 150;
+$yiq-contrasted-threshold:  150;
```

Comment:
whitespace te veel

Reply:
is nu gealigneerd met boven- & onderstaande variabelen ;)